zipkin-redis/src/test/scala/com/twitter/zipkin/storage/redis/RedisIndexSpec.scala

```diff
@@ -55,56 +56,50 @@ class RedisIndexSpec extends RedisSpecification {
     List(ann1, ann2), List(binaryAnnotation("BAH2", "BEH2")))
 
   test("index and get span names") {
-    result(redisIndex.indexSpanNameByService(span1))
+    ready(redisIndex.index(span1))
     result(redisIndex.getSpanNames("service")) should be (Set(span1.name))
   }
 
   test("index and get service names") {
-    result(redisIndex.indexServiceName(span1))
+    ready(redisIndex.index(span1))
     result(redisIndex.getServiceNames) should be (Set(span1.serviceNames.head))
   }
 
-  test("index only on annotation in each span with the same value") {
-    result(redisIndex.indexSpanByAnnotations(span3))
-  }
-
   test("getTraceIdsByName") {
-    var ls = List[Long]()
-    result(redisIndex.indexTraceIdByServiceAndName(span1))
-    result(redisIndex.getTraceIdsByName("service", None, 0, 3)) foreach {
```

Comment:
this and below it silently failed based on trying to have a timestamp comparison with zero!